integration-tests/search.test.ts

```diff
@@ -117,6 +117,46 @@ describe('search', () => {
         }
     });
 
+    test('tokens', async () => {
+        const randomPatientData = randomPatient();
+        randomPatientData.identifier = [
+            {
+                system: 'http://fwoa-integ-tests.com',
+                value: 'someCode',
+            },
+        ];
+        const testPatient: ReturnType<typeof randomPatient> = (await client.post('Patient', randomPatientData)).data;
+
+        const randomPatientDataNoSystem = randomPatient();
+        randomPatientData.identifier = [
+            {
+                value: 'someCodeWithoutSystem',
```

Comment:
Which test cases below test this insert?

Reply:
it is used on the test on L157. but there was an error on the var name, it should be `randomPatientDataNoSystem.identifier`. It's fixed now